lib/go-tc/federation_resolver.go

```diff
@@ -25,6 +25,11 @@ import "net"
 
 import "github.com/go-ozzo/ozzo-validation"
 
+type FederationResolverResponse struct {
```

Comment:
GoDoc?

Also, to avoid breaking changes to old client call signatures, we've started explicitly versioning library response structures with a rolling alias for the latest minor version. So in this case it'd look like

```go
type FederationResolverResponse struct {
    Alerts
    Response FederationResolver `json:"response"`
}

type FederationResolverResponseV4 = FederationResolverResponseV40
```
and then the client uses the V4 version

Reply:
Versioning the `FederationResolverResponse` struct in 2532a74f0a and added godocs.